package.json

```diff
@@ -13,12 +13,12 @@
   "author": "tribe29 GmbH",
   "license": "GPL-2.0-only",
   "devDependencies": {
-    "@grafana/data": "^8.5.4",
-    "@grafana/runtime": "^8.5.4",
-    "@grafana/toolkit": "^8.5.4",
-    "@grafana/ui": "^8.5.4",
+    "@grafana/data": "^9.0.4",
+    "@grafana/runtime": "^9.0.4",
+    "@grafana/toolkit": "^9.0.4",
+    "@grafana/ui": "^9.0.4",
     "@testing-library/jest-dom": "5.16.4",
-    "@testing-library/react": "^12.1.5",
+    "@testing-library/react": "^13.3.0",
```

Comment:
We could think about removing these two for now, as we don't use them currently and probably won't do so in the near future.

Reply:
removed them, thx!